tests/ludwig/datasets/mnist/test_mnist_workflow.py

```diff
@@ -0,0 +1,72 @@
+import os
+import unittest
+from unittest import mock
+from unittest.mock import mock_open
+import shutil
+from ludwig.datasets.mnist import Mnist
+
+"""TODO: This serves as a placeholder integration test, will be changed into a true unit test soon
+with mocking and patching, commented out for now"""
+
+
+class TestMNistWorkflow(unittest.TestCase):
```

Comment:
Instead of using `unittest`, we should follow the convention in the other tests and just use `pytest` fixtures.

Reply:
Fixed